pages/index.vue

```diff
@@ -1,3 +1,35 @@
 <template>
-    <h1>Hello world!</h1>
-</template>
\ No newline at end of file
+  <div class="my-team">
+    <header class="my-team__main-title">
+      <h1>The Creative crew</h1>
+    </header>
+    <div class="my-team__description">
+      <h2>Who we are</h2>
+      <p>
+        We are a team of creatively diverse. driven. innovative individuals
+        working in various locations from the world
+      </p>
+    </div>
+    <MyTeamMembers />
+  </div>
+</template>
+<script>
+export default {
+  name: 'TeamPage',
+  data() {
+    return {
+      members: [],
+    };
+  },
+  async fetch() {
+    this.members = await fetch(
+      process.env.API_BASE_URL + '/members/'
```

Comment:
thoughts on doing this instead?

`${process.env.API_BASE_URL}/members/`

Reply:
the fetch here is not being used.